wordpress-mobile/WordPress-iOS
# Comment Detail Redesign: Moderation Bar Update

Fixes #18999

This PR removes the old moderation bar from comments section and adds a new table-view version of it.

To test:
1. Log in to a WP Account
2. In My Site, select Menu
3. Comments
4. Select any comment
5. Check if the old moderation bar has been removed.
6. Validate if a new section called "STATUS" has been added below and it is responsive to selections.

## Regression Notes
1. Potential unintended areas of impact
Comments' status may not function properly

7. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested it. Follow the steps above.,

8. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
